source/jormungandr/jormungandr/scenarios/helper_classes/transfer.py

```diff
@@ -167,9 +192,16 @@ def _do_no_access_point_transfer(self, section):
     def _aysnc_no_access_point_transfer(self, section):
         return self._future_manager.create_future(self._do_no_access_point_transfer, section)
 
-    def _get_access_points(self, stop_point_uri, access_point_filter=lambda x: x):
-        sub_request_id = "{}_transfer_start_{}".format(self._request_id, stop_point_uri)
-        stop_points = self._instance.georef.get_stop_points_from_uri(stop_point_uri, sub_request_id, depth=2)
+    def _get_access_points(self, pt_object, access_point_filter=lambda x: x):
```

Comment:
there should be a more proper way to do it, you can create an abstract function in PtPlanner that returns access points, and the implementation should be "hidden" in the concrete class base on the type of PtPlanner,